src/backend/marsha/core/tests/test_api_liveregistration.py

```diff
@@ -0,0 +1,790 @@
+"""Tests for the LiveRegistration API."""
+from datetime import timedelta
+import json
+import random
+
+from django.db.utils import IntegrityError
+from django.test import TestCase
+from django.utils import timezone
+
+from rest_framework_simplejwt.tokens import AccessToken
+
+from ..factories import (
+    ConsumerSiteFactory,
+    LiveRegistrationFactory,
+    PlaylistFactory,
+    VideoFactory,
+)
+from ..models import LiveRegistration
+
+
+class LiveRegistrationApiTest(TestCase):
+    """Test the API of the liveRegistration object."""
+
+    def test_api_liveregistration_read_detail_anonymous(self):
+        """Anonymous users should not be allowed to retrieve a liveregistration."""
+        video = VideoFactory()
+        liveregistration = LiveRegistrationFactory(
+            email="chantal@test-fun-mooc.fr", video=video
+        )
+        response = self.client.get(
+            "/api/liveregistrations/{!s}/".format(liveregistration.id)
```

Comment:
```suggestion
            f"/api/liveregistrations/{liveregistration.id}/"
```

The new rule `consider-using-f-string` in pylint 2.11 will warn you about usage of `format` and will ask you to change to `f-string`.